accord-core/src/main/java/accord/local/Command.java

```diff
@@ -666,6 +672,11 @@ public final boolean isTruncated()
         return status().hasBeen(Status.Truncated);
     }
 
+    public static boolean isTruncated(Status status)
+    {
+        return status.hasBeen(Status.Truncated);
```

Comment:
```
Truncated         (Cleanup, Maybe, DefinitionErased,  ExecuteAtErased,  DepsErased,   Outcome.Erased),
Invalidated       (Persist, Maybe, NoOp,              NoExecuteAt,      NoDeps,       Outcome.Invalidated),
```

wouldn't that include `Invalidated`?

Reply:
So I have simply copied the code from above; my intention was to make non-static version use the static one as I did with `isStable`. You are right `isTruncated` is included here.